### PR TITLE
Add the unit test on expectation of distribution

### PR DESCRIPTION
**Pull request by hbgxsm95**
Finish one TODO

### CODE REVIEW

File: stats_test.go [hunk 1]
```diff
@@ -32,6 +32,8 @@ import (
 	monitoredrespb "google.golang.org/genproto/googleapis/api/monitoredres"
 	monitoringpb "google.golang.org/genproto/googleapis/monitoring/v3"
 	"google.golang.org/grpc"
+	"google.golang.org/genproto/googleapis/api/distribution"
+	"fmt"
```
Comment (ramonza) [inline stats_test.go]: run goimports to fix the import order (standard library should be at the top)

File: stats_test.go [hunk 2]
```diff
@@ -335,6 +377,8 @@ func TestExporter_makeReq(t *testing.T) {
 				return
 			}
 			if !reflect.DeepEqual(resps, tt.want) {
+				temp := reflect.DeepEqual(resps, tt.want)
```
Comment (ramonza) [inline stats_test.go]: was this accidentally left in?

**Comment by songy23** [comment]
Yes, please fix the format and remove the debugging sentences.

File: stats_test.go (hunk 2 above)
Comment (hbgxsm95) [reply stats_test.go]: Yes.  This code is for debugging.  I would remove it.

File: stats_test.go (hunk 1 above)
Comment (hbgxsm95) [reply stats_test.go]: Okay got it.

**Pull request merged**